src/components/CompletedTask.jsx

```diff
@@ -1,45 +1,41 @@
 import { ActionIcon, Card, Group, Text } from "@mantine/core";
 import { IconCircleCheck, IconCircleMinus } from "@tabler/icons-react";
-import React from "react";
 
-class CompletedTask extends React.Component {
-  render() {
-    const { task, handleDelete, handleToggleIsCompleted } = this.props;
-    return (
-      <Card bg="gray.2" px="xl" py="xs" mx="xl" withBorder>
-        <Group position="apart">
-          <Text
-            weight={500}
-            size={"15px"}
-            color="gray.6"
-            td="line-through"
-            maw={"70%"}
+const CompletedTask = ({ task, handleDelete, handleToggleIsCompleted }) => {
```

Comment:
[nitpick] Instead of naming handleThis, handleThat. Name methods with the action that it is doing. In this case handleDelete I believe is deleting a task. Could be renamed to deleteTask.